sw/device/tests/crt_test.c

```diff
@@ -54,6 +58,91 @@ static void init_uart(void) {
   base_uart_stdout(&uart0);
 }
 
+static void test_crt_section_clear(void) {
+  // Maximum end index of target section.
+  const size_t len = 32;
+
+  // Section indices (start inclusive, end exclusive).
+  struct {
+    size_t start;
+    size_t end;
+  } tests[6] = {{.start = 0, .end = 0},         {.start = 0, .end = 1},
+                {.start = len - 1, .end = len}, {.start = 0, .end = len - 1},
+                {.start = 1, .end = len},       {.start = 0, .end = len}};
+
+  for (size_t t = 0; t < sizeof(tests) / sizeof(tests[0]); ++t) {
```

Comment:
You can use ARRAYSIZE from memory.h here.

Reply:
Thanks, done.